app/components/Admin/AddIntake.tsx

```diff
@@ -171,11 +230,28 @@ const AddIntake: React.FC<Props> = ({ applicationQuery }) => {
       errors?.endDate.addError('End date & time must be entered');
     }
 
-    if (startDateTime <= currentDateTime) {
+    if (isEdit && nextIntake && endDateTime >= nextIntakeStartDateTime) {
+      errors?.endDate.addError(
+        'End date & time must not overlap with the next intake'
+      );
+    }
+
+    if (
+      (!isEdit && startDateTime <= currentDateTime) ||
+      // only check start date is in the future if editing an intake that has not started yet and does not have start date locked
+      (isEdit &&
+        currentDateTime <= intakeStartDate &&
+        startDateTime <= currentDateTime)
+    ) {
       errors?.startDate.addError(
         'Start date & time must be after current date & time'
       );
-    } else if (startDateTime <= latestIntakeEndDateTime) {
+    } else if (
+      // check latest intake start date overlap if creating a new intake
+      (startDateTime <= latestIntakeEndDateTime && !isEdit) ||
+      // check previous intake end date overlap if editing an intake
+      (isEdit && startDateTime <= previousIntakeEndDateTime)
```

Comment:
I'd argue that could cause issues with technically having 2 intakes open at the same time (if only for a minute). Starting the next intake at `2:31` would make more sense I think? 🤔

Reply:
If the `seconds` is set to 0 it (should) be an instant switchover, but I agree there should be some sort of more clear separation especially since we're not displaying the `seconds` value.